R/meta_regression/regression_plot_panel.R

```diff
@@ -211,6 +221,12 @@ regression_plot_panel_ui <- function(id) {
 regression_plot_panel_server <- function(id, data, covariate_title, covariate_name, model_output, treatment_df, outcome_type, outcome_measure, reference, package = "gemtc") {
   shiny::moduleServer(id, function(input, output, session) {
     
+    # This will show a spinner while the model calculates, but will not render anything once it has completed
+    output$spinner <- renderUI({
+      model_output()
+      NULL
```

Comment:
Out of interest, what does NULL do here?

Reply:
It returns null to the render UI so that it renders nothing. I tried to add a spinner without doing this, but I couldn't get the events to trigger in the correct order.